src/Microsoft.AspNet.OData.Shared/Query/Expressions/AggregationBinder.cs

```diff
@@ -48,7 +48,9 @@ internal class AggregationBinder : ExpressionBinderBase
             {
                 case TransformationNodeKind.Aggregate:
                     var aggregateClause = this._transformation as AggregateTransformationNode;
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
don't use #pragma. 

I think you can change the codes as:

_aggregateExpressions = FixCustomMethodReturnTypes(aggregateClause.AggregateExpressions.OfType<AggregateExpression>());

Reply:
That's correct. In https://github.com/OData/WebApi/pull/906/files#diff-b18fa1821acbe9c329e386961320ad35 I'm changing fields to accept base class or using OfType